README.md

```diff
@@ -31,6 +31,18 @@ observability tools.
 [Jaeger]: https://www.jaegertracing.io
 [msrv]: #supported-rust-versions
 
+
+## Project Status
+
+| Signal  | Status     | Project |
+| ------- | ---------- | ------- |
+| Traces  | Beta       | N/A     |
+| Metrics | Alpha      | N/A     |
+| Logs    | N/A    [1] | N/A     |
+
+- [1]: The Logs signal development is halted for this project while we develop both Traces and Metrics.
```

Comment:
I think we have one PR to add logs. So we probably shouldn'd say we won't accept the log PR